openapi_python_client/templates/property_templates/model_property.py.jinja

```diff
@@ -8,6 +8,8 @@
 {{ construct_template(construct_function, property, source, initial_value=initial_value) }}
 {% endmacro %}
 
+{% macro check_type_for_construct(property, source) %}isinstance({{ source }}, dict){% endmacro %}
+
 {% macro transform(property, source, destination, declare_type=True) %}
```

Comment:
We probably need a better name for this macro than "transform" at some point 😅even I can't remember what this one is for sometimes. Not a note on this PR, just a note on how I need to put some design thought into the templates.

Reply:
Haha, yeah...maybe `serialize` and `deserialize`?

Reply:
or `to_json` and `from_json`...